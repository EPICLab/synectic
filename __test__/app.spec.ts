import Welcome from '../src/components/welcome';

describe('Welcome', () => {
  let instance: Welcome;

  beforeEach(() => {
    instance = new Welcome({});
  });

  it('creates an instance of App', async () => {
    expect(instance).toBeInstanceOf(Welcome);
  });

  it(`testing snapshots match`, () => {
    const bar = {
      foo: {
        x: 2,
        y: 2
      },
    };

    expect(bar).toMatchSnapshot();
  });
})