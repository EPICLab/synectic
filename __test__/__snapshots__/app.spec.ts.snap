// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome testing snapshots match 1`] = `
Object {
  "foo": Object {
    "x": 1,
    "y": 2,
  },
}
`;
